rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/service/ConfigManagementServiceImpl.java

```diff
@@ -186,12 +186,15 @@ public void removeConnectorConfig(String connectorName) {
         ConnectKeyValue config = new ConnectKeyValue();
         config.put(RuntimeConfigDefine.UPDATE_TIMESATMP, System.currentTimeMillis());
         config.put(RuntimeConfigDefine.CONFIG_DELETED, 1);
-        Map<String, ConnectKeyValue> connectorConfig = new HashMap<>();
-        connectorConfig.put(connectorName, config);
         List<ConnectKeyValue> taskConfigList = new ArrayList<>();
         taskConfigList.add(config);
 
         connectorKeyValueStore.put(connectorName, config);
+        try {
+            putConnectorConfig(connectorName, config);
```

Comment:
@imaffe Could you please give more details about where and how does putConnectorConfig() function put a tombstone record? My understanding here is that `putConnectorConfig(connectorName, config); ` will probably only reload jdbc connector.

Reply:
2 line here:
```
config.put(RuntimeConfigDefine.UPDATE_TIMESATMP, System.currentTimeMillis());
config.put(RuntimeConfigDefine.CONFIG_DELETED, 1);
```

and it overwrites the connectors config and set the CONFIG_DELETED bit to 1.